generator/lib/builder/om/QueryBuilder.php

```diff
@@ -670,7 +670,7 @@ protected function addFilterByPrimaryKey(&$script)
      *
      * @return " . $this->getStubQueryBuilder()->getClassname() . " The current query, for fluid interface
      */
-    public function filterByPrimaryKey(\$key)
+    public function filterByPrimaryKey(\$key, \$comparison = Criteria::EQUAL)
     {";
```

Comment:
Nice patch ;)
Just a minor suggestion:
add the new "@param     string $comparison" in the description of this method ;)